airbyte-integrations/connectors/source-github/acceptance-test-config.yml

```diff
@@ -3,143 +3,139 @@ test_strictness_level: "high"
-          deployments: ["airbytehq/integration-test", "updated_at"]
-          events: ["airbytehq/integration-test", "created_at"]
-          issue_comment_reactions:
-            ["airbytehq/integration-test", "907296275", "created_at"]
-          issue_events: ["airbytehq/integration-test", "created_at"]
-          issue_milestones: ["airbytehq/integration-test", "updated_at"]
-          issue_reactions: ["airbytehq/integration-test", "created_at"]
-          issues: ["airbytehq/integration-test", "updated_at"]
-          project_cards:
-            ["airbytehq/integration-test", "13167124", "17807006", "updated_at"]
-          project_columns:
-            ["airbytehq/integration-test", "13167124", "updated_at"]
-          projects: ["airbytehq/integration-test", "updated_at"]
-          pull_request_comment_reactions:
-            ["airbytehq/integration-test", "699253726", "created_at"]
-          pull_request_stats: ["airbytehq/integration-test", "updated_at"]
-          pull_requests: ["airbytehq/integration-test", "updated_at"]
-          releases: ["airbytehq/integration-test", "created_at"]
-          repositories: ["airbytehq", "updated_at"]
-          review_comments: ["airbytehq/integration-test", "updated_at"]
-          reviews: ["airbytehq/integration-test", "updated_at"]
-          stargazers: ["airbytehq/integration-test", "starred_at"]
-          workflow_runs: ["airbytehq/integration-test", "updated_at"]
-          workflows: ["airbytehq/integration-test", "updated_at"]
-          workflow_jobs: ["airbytehq/integration-test", "completed_at"]
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      future_state:
+        future_state_path: "integration_tests/abnormal_state.json"
+      cursor_paths:
+        comments: ["airbytehq/integration-test", "updated_at"]
+        commit_comment_reactions: ["airbytehq/integration-test", "55538825", "created_at"]
+        commit_comments: ["airbytehq/integration-test", "updated_at"]
+        commits: ["airbytehq/integration-test", "master", "created_at"]
+        deployments: ["airbytehq/integration-test", "updated_at"]
+        events: ["airbytehq/integration-test", "created_at"]
+        issue_comment_reactions: ["airbytehq/integration-test", "907296275", "created_at"]
+        issue_events: ["airbytehq/integration-test", "created_at"]
+        issue_milestones: ["airbytehq/integration-test", "updated_at"]
+        issue_reactions: ["airbytehq/integration-test", "created_at"]
+        issues: ["airbytehq/integration-test", "updated_at"]
+        project_cards: ["airbytehq/integration-test", "13167124", "17807006", "updated_at"]
+        project_columns: ["airbytehq/integration-test", "13167124", "updated_at"]
+        projects: ["airbytehq/integration-test", "updated_at"]
+        pull_request_comment_reactions: ["airbytehq/integration-test", "699253726", "created_at"]
```

Comment:
These are just getting back on the same line after allowing for longer line length